scripts/release.sh

```diff
@@ -20,7 +20,7 @@ elif [ "$RELEASE_TYPE" == "release" ] && [ "$CURRENT_BRANCH" == "main" ]; then
 fi
 
 for pkg in $PKG_NPM_LABELS ; do
-  bazel run --config=release -- ${pkg}.publish --access public --tag ${NPM_TAG}
+  bazel run -- ${pkg}.publish --access public --tag ${NPM_TAG}
```

Comment:
Is it expected that the `--config=release` flag will always be remembered? I'm picturing a scenario where someone wants to publish locally and forgets that flag. Or, do we want to just say publishing is _always_ done on CI? Maybe even have some condition at the top of this script that checks for some CI-only env var and fails our if not? Seems like maybe overkill, but just a thought I had.

Reply:
I could see people running individual commands if needed, but I wouldn't envision anyone running the entire `release` script locally. 

Enforcing that from a CI env var isn't a bad idea